elixir/config/config.exs

```diff
@@ -212,7 +214,7 @@ config :phoenix, :json_library, Jason
 
 config :swoosh, :api_client, Swoosh.ApiClient.Finch
 
-config :web, Web.Mailer,
```

Comment:
This should be moved to domain section of the config (see `##### ... ` delimiters).

Reply:
Are you saying you want this entire `config` block moved up to the `Domain` section even though it was here when it was in the `Web` section?